### ISSUE TITLE
 Microsoft 365 learning pathways has a configuration issue. Ask your administrator for assistance.

### ISSUE THREAD

**Issue by actuallygob** [pnp/custom-learning-office-365#227]

**Comment by juliemturner** [comment]
@actuallygob So it looks like you're new to GitHub, welcome.  You may want to check out the M365 PnP Sharing is Caring initiative to get up to speed on using the platform. It's definitely helpful if you're just getting started. https://aka.ms/sharing-is-caring.  For future reference when you're creating an issue please pick Bug Report, Enhancement, or Question and fill out the template as completely as you can, this helps me help you faster.  Also please don't respond to the thread via email as doing so puts email headers and PII in the body of the issue exposing things you may not want to be exposed publically.

Onto your issue, it appears based only on the title that you're having problems with the solution.  Unfortunately, you didn't really give me any details in the body of the issue. That message often has to do with the configuration of your environment and/or a proxy blocking the web page from accessing the location where the manifests are located.

If you could please provide a screenshot of where the issue is happening and also the browsers development tools console log that would be helpful.  If you're non-technical and can ask someone in your IT department for assistance that might make things easier. However, the browser console can most often be reched by pressing the F12 key. Then refresh the page. Check out the "Console" and paste any errors that it reports in the issue so that I can point you in the right direction.

**Comment by zoryatix** [comment]
Hi, 

Jumping in on this one since I've just had the same error message.

The https://provisioning.sharepointpnp.com/ kept failing for me.
So, I was following your manual installation guide found here: https://github.com/pnp/custom-learning-office-365/blob/master/webpart/README.md

This part is where I am getting the error:
https://github.com/pnp/custom-learning-office-365/blob/master/webpart/README.md#initialize-web-part-custom-configuration
Basically, trying to access "https://(TenantName).sharepoint.com/sites/(SiteCollectionName)/SitePages/CustomLearningAdmin.aspx"

I have a log of all the error messages from the browser console. Not sure if I should post it here, since I don't know if it contains any sensitive info.

**Comment by juliemturner** [comment]
You may have the same message but it may not be for the same reason, so ideally you'd want to open your own issue. 

That said since the other person has not responded I can try and assist you. In general, there isn't really anything particularly sensitive in the console log but to make it simpler if you want to send me the details of the console you can upload them here: https://symp.info/M365LPIssue 

>Please put your file in the folder with the correlating issue number, in this case 227 and then respond here that your file has been shared.  After diagnoses I'll delete it.

**Comment by shelagh03** [comment]
I have just done exactly the same and am getting the same error. Happy to share as much information as you need here or open the same report in a fresh post.
Please let me know what you would prefer?

**Comment by juliemturner** [comment]
Sure go ahead and submit your details as I instructed since no one else is following through.

**Comment by shelagh03** [comment]
I think we may be in different time zones - apologies for the delay!
I have uploaded what I see in the console - I hope that is what you need? Let me know if not
Thanks for your help

**Comment by juliemturner** [comment]
@shelagh03 -- So I see this error: 

> Cannot read property 'Manifest' of undefined - DataService (refreshConfig) 

This implies that it was not able to download the metadata.json file from the GitHub repo. That said you seem to have some other customizations loading that are throwing errors so it's a bit confusing to review.  Anyway, can you please review the network tab of the browser dev tools. What you want to do is search for metadata.json and then see if you're successfully downloading the file. If not it's either that somehow your CDN endpoint isn't correct or that you have network configuration that is blocking your ability to download content from GitHub.

**Comment by shelagh03** [comment]
@juliemturner I have searched for metadata.json in the Network tab and it did not return any results.
Should I try re-installing do you think?

**Comment by juliemturner** [comment]
Reinstalling will not help as this is not an installation issue.  That it doesn't appear in the network tab only means that you didn't have the development tools open when the page loaded.  Open the development tools and then reload the page. Metadata.json will appear in the list of assets downloaded although may have an error condition.

**Comment by shelagh03** [comment]
Thanks - done it again but not quite sure how to share the result as it wont copy and paste. I've taken a screen snip - let me know if it is accessible here. If not I can upload it to the files area 
![metadata_json](https://user-images.githubusercontent.com/39377752/77596397-c76ae080-6ef3-11ea-8ec8-d58f3eba8aff.PNG)

**Comment by juliemturner** [comment]
Um, not sure where you're at but that is not what I'm looking for.  Here's a screen shot of what you should see (this is chrome but it should be similar in other browsers)

![image](https://user-images.githubusercontent.com/7570936/77596797-629f8e00-6ed3-11ea-8fb8-14d03cf07509.png)

**Comment by shelagh03** [comment]
![image](https://user-images.githubusercontent.com/39377752/77597432-c5565100-6ef6-11ea-99db-cd1cdb6ce3d8.png)
Sorry - clearly I'm doing something wrong her - this is a screenshot

**Comment by juliemturner** [comment]
Yep, you need to click the filter icon and then type metadata into the filter.

![image](https://user-images.githubusercontent.com/7570936/77649663-9c5aae00-6f40-11ea-8e08-362ca4f49c4e.png)

**Comment by juliemturner** [comment]
Given the difficulty I've written up some guidance as a starting point for diagnosing this issue with screen shots.  Hopefully that helps everyone. https://github.com/pnp/custom-learning-office-365/blob/master/webpart/DiagnosisPreCheck.md

**Comment by shelagh03** [comment]
Thanks for the detailed guidance which I have followed to produce a console log in the correct way! I have uploaded this to the folder you specified above. 
A colleague suggested that I generate a HAR.file so that you could reproduce exactly what I am seeing so I have done that and also uploaded it in the same place.
I have also checked for metadata.json in the network tab using the filter (rather than search) and can confirm that it is definitely not loading. 
Unless you can see something obvious I wonder whether I should do as you suggested in a more recent post and completely remove this install and start again? The original install was last June so maybe there is some incompatibility between that install and the upgraded file?
Thanks for your patience! @juliemturner

**Comment by juliemturner** [comment]
Glad you're all set. I'm going to close this and if anyone else that responded to this thread has a similar issue please follow the guidelines suggested in this [article](https://github.com/pnp/custom-learning-office-365/blob/master/webpart/DiagnosisPreCheck.md) and if still having an issue start a new issue.
